include/uapi/linux/r_taurus_cluster_protocol.h

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#ifndef R_TAURUS_CLUSTER_PROTOCOL_H
+#define R_TAURUS_CLUSTER_PROTOCOL_H
+
+#ifndef __packed
+# define __packed       __attribute__((__packed__))
+#endif
+
+#include <linux/r_taurus_bridge.h>
```

Comment:
I am not sure that this include will be available to userpsace. Could you please check?

Reply:
this is because it took the file from rvgc :( and did not check